app/assets/stylesheets/pages.css.scss

```diff
@@ -104,6 +104,15 @@
       margin-bottom: 8px;
     }
 
+    .nickname {
+      display: inline-block;
+    }
+
+    .time {
+      display: inline-block;
+      color: gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.